### ISSUE TITLE
Set Export file name for ipynb file

### ISSUE THREAD

**Issue by shrysr** [jkitchin/scimax#198]
I am able to use the export file name property and correctly obtain the export file name in the desired location - for all exports except ipython notebook export. 

Using the ipython notebook option creates the file with the same name as the source org file.

**Comment by jkitchin** [comment]
This is currently not a supported option for ipython notebooks.  It might be feasible to add some limited support for it. How do you normally use this feature for other exports?

**Comment by shrysr** [comment]
I'd like to control the content of different projects on ipython notebooks from my notes in org, rather than editing in Jupyter notebooks. So I'd need to export a org-subheading to the appropriate location in a git repository. One possibility would be to use separate org files for each project and place it at the ipynb notebook location - I'd rather not to do this as it breaks the cohesiveness of my notes.

At the moment, frequent exports for me are hugo blog posts from org notes, for which I use a modified version of : http://www.holgerschurig.de/en/emacs-blog-from-org-to-hugo/ . These use a separate property for the HUGO_FILENAME, where I use the org-id.  

With the other exports, I am able to use the EXPORT_FILE_NAME property under the org subheading. Hope that answers your question? Would be glad for options I could explore.

**Comment by shrysr** [comment]
I have updated my git submodules using `git submodule update --remote --merge` . Restarted scimax daemon - but the problem persists. The subtree is exported with the same name as the org file in the case of ipynb exports.

The feedback from the terminal during the update is as follows : 
```
Updating bc61d4d..9bbf2fa
Fast-forward
 org-mime.el | 241 ++++++++++++++++++++++++++++++++++++++++++++++++------------
 1 file changed, 193 insertions(+), 48 deletions(-)
Submodule path 'org-mime': merged in '9bbf2faaddb7069bba0d3dc9134c3edd1705c834'
Updating f833b79..c5812f6
Fast-forward
 doi-utils.el           | 141 +++++++++++++++++++++++++------------------------
 org-ref-bibtex.el      |   8 +--
 org-ref-core.el        |  62 +++++++++++++++++++---
 org-ref-helm-bibtex.el |   2 +-
 org-ref-helm.el        |   4 +-
 org-ref-utils.el       |  22 ++++----
 org-ref.el             |   4 +-
 7 files changed, 150 insertions(+), 93 deletions(-)
Submodule path 'org-ref': merged in 'c5812f6da756f58342f9cdff380ade8dca9e56e1'
Updating 29e4378..8b9b501
Fast-forward
```

**Comment by jkitchin** [comment]
glad to hear it works. I will go ahead and close this.
